features/step_definitions/operator.rb

```diff
@@ -9,3 +9,125 @@
   })
   step %Q/the step should succeed/
 end
+
+Given /^etcd operator "([^"]*)" is removed successfully from "([^"]*)" project$/ do | name, proj_name|
```

Comment:
I would like to have a sub-directory `upgrade` under `step_defnitions` where steps only useful for upgrade testing will be held.

Reply:
OK, I will modify it. Thanks!